### DISCUSSION TITLE
Universe '23 Day 2 Keynote Watch Party: The productivity platform for all developers [DEV2845K]

### DISCUSSION THREAD

**Discussion by jricciardi** [community/community#74856] [Other Feature Feedback, Questions, & Ideas]
![image](https://github.com/community/community/assets/31521880/ba36c23b-1b84-4f73-b583-cdb5c849fdcb)

As the technology landscape evolves, GitHub is your strategic partner to help you securely apply AI towards human progress, no matter your developer skillsets or enterprise readiness. To prepare you for tomorrow, GitHub is the productivity platform for all developers.

**Ways to watch**
* Watch the [Recording on YouTube](https://www.youtube.com/watch?v=h_o9kFPVeiw&ab_channel=GitHub)

* [Streaming on GitHubUniverse.com ](https://reg.githubuniverse.com/flow/github/universe23/attendee-portal/page/welcome) 
   * The portal contains recorded sessions from yesterday, the full session & speaker catalog, and other handy resources
* [Streaming on YouTube](https://www.youtube.com/watch?v=hFCL3B7kRds&ab_channel=GitHub)
* [Streaming on LinkedIn](https://www.linkedin.com/company/github/posts/?feedView=all)

**Session Speakers**

- Inbal Shani, Chief Product Officer, GitHub
- Shubha Rao, VP, Product Strategy, GitHub
- Asha Chakrabarty, VP, Product Management, GitHub
- Stormy Peters, VP, Communities, GitHub
- Gerard McMahon, Head of ALM Tools and Platforms, Fidelity Investments

Catch **[yesterday's Day 1 keynote here](https://www.youtube.com/watch?v=NrQkdDVupQE&ab_channel=GitHub)**

Check out all the biggest announcements over on the **[official GitHub Universe 2023 blog post](https://github.blog/2023-11-08-universe-2023-copilot-transforms-github-into-the-ai-powered-developer-platform/)**

Join in the conversation

* And don't forget to get your [Universe keynote BINGO card](https://docs.google.com/document/d/1poAUw3VIcV-utaxlVWw2BnqmDWdEmXJE2a4dDfJRHLc/edit) and play along 😄 
* [What was the first thing you ever developed on GitHub?](https://github.com/orgs/community/discussions/74871)
* [Just for fun: drop your favorite developer memes here! 🧵](https://github.com/orgs/community/discussions/74905)

**Comment by jricciardi** [comment]
Welcome to the stage, Inbal Shani!

![Screenshot 2023-11-09 at 9 03 45 AM](https://github.com/community/community/assets/31521880/d1ead96d-2711-4e68-b137-ef7b9f8dbbdd)



Inbal Shani
Chief Product Officer, GitHub

>As Chief Product Officer at GitHub, Inbal is committed to empowering developers by creating products and services that enhance developer experience and productivity. Prior to joining GitHub, she led engineering and product teams at Amazon and Microsoft, focusing on the development of enterprise and consumer-facing services. In her most recent position, Inbal was the General Manager of AWS Elastic Containers, where she was responsible for developing containers-based solutions to assist developers in modernizing their application development. She enjoys spending her spare time with her husband, three children, and Havanese puppy, running between swimming, cross-country, and basketball meets. Additionally, she enjoys drawing and painting.

**Reply by lclindeman** [reply]
👋

**Reply by jricciardi** [reply]
👋

**Reply by mishmanners** [reply]
👋

**Comment by jricciardi** [comment]
Green room sneak peak of a few other speakers coming on stage soon 👀 
![img_0482_720](https://github.com/community/community/assets/31521880/1774adab-5ade-4d06-b36d-f30c86a05810)

**Comment by jricciardi** [comment]
Welcome back to the main stage, Asha!

![Screenshot 2023-11-09 at 9 08 31 AM](https://github.com/community/community/assets/31521880/0d5825c7-1aef-4a36-918d-093a50904f3b)

Asha Chakrabarty
VP, Product Management, GitHub

> Asha Chakrabarty is the Vice President of Product Management for Security & Compute products at GitHub. Prior to joining GitHub, Asha was at Amazon Web Services (AWS) for 9+ years and wore multiple hats. Most recently at AWS, she led a global team of GTM and Product Specialists for AWS’ Container Services portfolio. Over the past 16+ years, her experience spans cloud computing, software development, modern application architectures, infrastructure design, and developer tools. Asha is based out of Austin, Texas where she spends her spare time with family, attempting to become a wine and dark chocolate connoisseur, and honing her novice photography skills.

**Reply by Akash1134** [reply]
🙌

**Comment by jricciardi** [comment]
Still totally mind blowing - 32 million security issues solved this YEAR ALONE along with GitHub Advanced Security 🤯 
![Screenshot 2023-11-09 at 9 11 15 AM](https://github.com/community/community/assets/31521880/2ff6ce93-37bd-4dd0-8aa2-e4c7a9bb092a)

**Comment by jricciardi** [comment]
If you didn't know: Dependency Scanning available cost free for open source

![Screenshot 2023-11-09 at 9 13 27 AM](https://github.com/community/community/assets/31521880/f49e4543-235d-45c9-9697-b0c05050534e)

**Comment by jricciardi** [comment]
Who was expecting this to be such a big applause line? 

![Screenshot 2023-11-09 at 9 18 47 AM](https://github.com/community/community/assets/31521880/ead53459-55f6-4fc6-b21d-c4a5e4812956)

**Reply by queenofcorgis** [reply]
Could hear it from the other side of the venue!!

**Reply by jricciardi** [reply]
That's awesome! ❤️

**Reply by mishmanners** [reply]
The video with @filmgirl was awesome too!

**Comment by jricciardi** [comment]
A new GitHub VP on the main stage - welcome, Shuba! (And dropping product reveals faster than I can post!)

![Screenshot 2023-11-09 at 9 21 15 AM](https://github.com/community/community/assets/31521880/c7fb5ce8-6ff4-45f6-9b51-be7e3daee1cf)

Shubha Rao
VP, Product Strategy, GitHub

> Shubha Rao works at GitHub as VP of Product focusing on Strategy, Platform, & Enterprise. Shubha brings experience in enabling new developer workflows and growing enterprise SaaS businesses from her years in AWS. She lives in Seattle with her husband and two kids. In her spare time, she enjoys traveling and hiking.

**Reply by jricciardi** [reply]
![Screenshot 2023-11-09 at 9 22 42 AM](https://github.com/community/community/assets/31521880/65843b7f-520b-4daa-a1b6-c834b2a1a6d7)

**Comment by jricciardi** [comment]
ACCOUNT SWITCHING IS LIVE TODAY. Heck yeah! Once-click to switch from work account to personal accounts securely 🙌 
![Screenshot 2023-11-09 at 9 24 36 AM](https://github.com/community/community/assets/31521880/0450555d-622d-43c0-ac42-9fd3b59ab7fe)

**Comment by jricciardi** [comment]
If you didn't know: Content Exclusions launching today — You can prevent specified files from being used to inform code completion suggestions made by GitHub Copilot ([Documentation here!](https://docs.github.com/en/copilot/managing-copilot-business/configuring-content-exclusions-for-github-copilot))
![Screenshot 2023-11-09 at 9 27 00 AM](https://github.com/community/community/assets/31521880/40928749-6f42-44da-96b7-71e28a5b6eac)

**Comment by jricciardi** [comment]
Welcome to Fidelity Investments' Head of ECC ALM Tools and Platforms, Gerard McMahon! On stage now for a chat with Inbal Shani all about how Fidelity Investments has been using GitHub Copilot.

![Screenshot 2023-11-09 at 9 30 43 AM](https://github.com/community/community/assets/31521880/90f3d54f-1b4d-4477-af0f-ad43d542474a)

Gerard McMahon
Head of ALM Tools and Platforms, Fidelity Investments

> Gerard McMahon is Head of ECC ALM Tools and Platforms at Fidelity Investments, whose mission is to substantially improve developer experience, reduce risk to the firm, improve feature velocity and drive greater end-to-end quality and efficiency. Ger a strong advocate of Software Delivery Excellence and incorporating the principles of agility, engineering and operational excellence in building high performing teams and enabling a DevOps culture. Ger is passionate about enabling connection and collaboration across the Software Delivery ecosystem, for enabling the breaking down of silo’s and creating data driven insights for driving innovation and continuous learning for the delivery acceleration of high quality business value.

**Comment by jricciardi** [comment]
Welcome back to the stage, Stormy! We're always excited to hear more about GitHub Sponsors 😍 
![Screenshot 2023-11-09 at 9 39 56 AM](https://github.com/community/community/assets/31521880/ff6e5934-e3ab-4e06-836f-efa491ba1e8c)

Stormy Peters
VP, Communities, GitHub

> Stormy Peters is VP of Communities at GitHub, where she leads the teams responsible for creating programs and services structured to enable online creators, developers, and open source communities. Throughout her career, Stormy has been passionate about open source software and has worked to educate companies and communities on how open source software is changing the software industry. Prior to GitHub, Stormy was the Director of the Open Source Programs Office at Microsoft, enabling 30,000+ developers to consume and contribute to open source effectively. She has worked at large organizations, started nonprofits, and led open source foundations. She currently serves on the Linux Foundation Board of Directors.

**Reply by jricciardi** [reply]
Fun announcements for open source: Like a Patreon integration with GitHub Sponsors - and 35 new regions supported for Sponsors (totaling 103 regions now!) 
![Screenshot 2023-11-09 at 9 42 23 AM](https://github.com/community/community/assets/31521880/18afcb3c-9ef5-4b65-aa4b-5acea86344a1)

**Reply by jricciardi** [reply]
Awesome behind the scenes shot! 
![IMG_0438](https://github.com/community/community/assets/31521880/aae2c4e8-67da-4c77-8188-fbe1cb8f97ae)

**Comment by lclindeman** [comment]
Read more about the GitHub Accelerator! https://github.blog/2023-11-02-celebrating-the-first-round-of-github-accelerator-and-whats-next/ 🚀

**Comment by Akash1134** [comment]
What a success this has been - [GitHub Accelerator](https://accelerator.github.com/) ✨ ANOTHER COHORT in 2024 🚀

**Reply by mishmanners** [reply]
Can't wait to see the next cohort!

**Comment by jricciardi** [comment]
Copilot is FREE for students, educators, and open source maintainers. Nearly half a million have taken advantage so far!

![Screenshot 2023-11-09 at 9 46 52 AM](https://github.com/community/community/assets/31521880/77107f10-31dd-4cc7-a27f-6be3fd1beffa)

**Comment by lclindeman** [comment]
"What I know about building is that it takes a team." 🙌

**Comment by jricciardi** [comment]
That's a wrap on the live keynote! We'll post a link to the keynote recording as soon as the recording is available, sometime later today 🎉

**Reply by jricciardi** [reply]
The recording is up! Watch the [recording on YouTube](https://www.youtube.com/watch?v=h_o9kFPVeiw&ab_channel=GitHub)
